triggers/github-trigger-event-sink/event.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+      "event_payload": {
```

Comment:
I'd imagine we'd want to be a bit more verbose here eventually. But this reaches parity so I am cool for now.